unlock-app/src/__tests__/components/creator/CreatorLockForm.test.js

```diff
@@ -246,7 +249,7 @@ describe('CreatorLockForm', () => {
       expect(wrapper.getByValue('35').dataset.valid).toBe('true')
     })
     it('max number of keys is infinity', () => {
-      const wrapper = makeLockForm({ maxNumberOfKeys: INFINITY })
+      const wrapper = makeLockForm({ maxNumberOfKeys: UNLIMITED_KEYS_COUNT })
```

Comment:
this should stay as INFINITY, that's the text symbol

Reply:
Nope. IO think we should only use INFINITY to display stuff, not as a value in store.

Reply:
I strongly think that `maxNumberOfKeys` should be a number, including one that represents infinity (we went fro -1 but could use something else if you think there is a better choice). So I think using `∞` is not a good approach.